project/Dependencies.scala

```diff
@@ -81,7 +81,7 @@ object Dependencies {
   // protobuf is only need to use the MySQL X DevAPI which we don't. exclude it to avoid interference with Google client libraries
   val mysqlConnector: ModuleID =  "com.mysql"                         % "mysql-connector-j"  % "9.0.0" exclude("com.google.protobuf", "protobuf-java")
   // Update warning for liquibase-core: Here be dragons! See https://broadworkbench.atlassian.net/browse/WOR-1197
-  val liquibaseCore: ModuleID =   "org.liquibase"                 % "liquibase-core"        % "4.17.2" // scala-steward:off
+  val liquibaseCore: ModuleID =   "org.liquibase"                 % "liquibase-core"        % "4.29.1"
```

Comment:
nit: remove the comment above this line

Reply:
We attempted to update liquibase last year and the checksum computation failed out. This is reintroducing that change. Against the checksums stored in the DEV DB we get the same error as last year (against a BEE), so I'm formulating a plan of attack to get this out the door.

Reply:
@davidangb see the PR description for my conclusions.